homeassistant/components/switchbot/strings.json

```
@@ -44,7 +44,7 @@
     },
     "abort": {
       "already_configured_device": "[%key:common::config_flow::abort::already_configured_device%]",
-      "no_devices_found": "[%key:common::config_flow::abort::no_devices_found%]",
+      "no_devices_found": "No supported SwitchBot devices found in range; If the device is in range, ensure the scanner has active scanning enabled, as SwitchBot devices cannot be discovered with passive scans. Active scans can be disabled once the device is configured. If you need clarification on whether the device is in-range, check diagnostics for the Bluetooth scanner to see if the MAC address of the SwitchBot device is present.",
```

```suggestion
      "no_devices_found": "No supported SwitchBot devices found in range; If the device is in range, ensure the scanner has active scanning enabled, as SwitchBot devices cannot be discovered with passive scans. Active scans can be disabled once the device is configured. If you need clarification on whether the device is in-range, download the diagnostics for the Bluetooth device to see if the MAC address of the SwitchBot device is present.",
```
Maybe "check diagnostics" might sound too vague?

---

We need them figure out that it's the diagnostics for the Bluetooth integration (or esphome or Shelly)

---

Maybe "for the integration that provides your Bluetooth adapter or proxy"?

---

Updated original suggestion